views/js/qtiCreator/editor/interactionsToolbar.js

```diff
@@ -74,6 +74,20 @@ define([
 
         _.each(interactions, function(interactionAuthoringData){
             add($sidebar, interactionAuthoringData);
+
+            const $interactionContainer = $(`li[data-qti-class="${interactionAuthoringData.qtiClass}"]`);
+            const $interactionText = $interactionContainer.find('div');
+
+            const interactionContainerWidth = $interactionContainer.width();
+            const interactionTextWidth = $interactionText.outerWidth(true);
+
+            if (interactionTextWidth >= interactionContainerWidth) {
+                const fontSize = 100 - (interactionTextWidth - interactionContainerWidth);
```

Comment:
**thought:** I wonder if this computation is robust enough. It takes a percentage and a size diff. What would happen if the width is bigger than 100? Should not it be better to rely on a ratio instead? Something like `100 * (size / maxSize)`

Reply:
Thanks for the check. PCI items have `<img>` instead of `<span>` for the icons. Now both are equal.
Refactor the computation too.